vhdbuilder/scripts/automate_release.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -euxo pipefail
+source vhdbuilder/scripts/automate_helpers.sh
+
+EV2_ARTIFACT_PIPELINE_ID="319030" 
+SIG_RELEASE_PIPELINE_ID="494"
+
+VHD_BUILD_ID="${VHD_BUILD_ID:-""}"
+
+trigger_ev2_artifacts() {
+    echo "creating EV2 artifacts for VHD build with ID: $VHD_BUILD_ID"
+
+    # Run the pipeline and fetch the run ID to poll for success later
+    EV2_BUILD_ID=$(az pipelines run --id $EV2_ARTIFACT_PIPELINE_ID --variables "VHD_PIPELINE_RUN_ID=$VHD_BUILD_ID" | jq -r '.id')
+    if [ $? -ne 0 ]; then
+        echo "failed to trigger EV2 artifact build"
+        return 1
+    fi
+
+    STATUS="$(az pipelines runs show --id $EV2_BUILD_ID | jq -r '.status')"
+    while [ "${STATUS,,}" == "notstarted" ] || [ "${STATUS,,}" == "inprogress" ]; do
+        echo "EV2 artifact build $EV2_BUILD_ID is still in-progress..."
+        sleep 60
+        STATUS="$(az pipelines runs show --id $EV2_BUILD_ID | jq -r '.status')"
+    done
+
+    if [ "${STATUS,,}" != "completed" ]; then
+        echo "EV2 artifact build failed for VHD build with ID: $VHD_BUILD_ID, failed build ID: $EV2_BUILD_ID"
+        return 1
+    fi
+    echo "EV2 artifacts successfully built for VHD build with ID: $VHD_BUILD_ID, EV2 build ID: $EV2_BUILD_ID"
+}
+
+create_release() {
+    echo "creating SIG release for VHD with build ID: $VHD_BUILD_ID"
+
+    EV2_BUILD_ID="94988139"
```

Comment:
can we add a URL in comment to allow for quick lookup

Reply:
this is currently just for testing